_overviews/scala3-book/methods-main-methods.md

```diff
@@ -81,7 +81,7 @@ Illegal command line: java.lang.NumberFormatException: For input string: "sixty"
 The Scala compiler generates a program from an `@main` method `f` as follows:
 
 - It creates a class named `f` in the package where the `@main` method was found.
-- The class has a static method `main` with the usual signature: It takes an `Array[String]` as argument and returns `Unit`.
+- The class has a static method `main` with the usual signature of Java main methods: it takes an `Array[String]` as argument and returns `Unit`.
```

Comment:
“Java main methods” could be changed to “Java `main` methods”.

Reply:
Fixed